server/server.go

```diff
@@ -54,6 +54,20 @@ func logger() *logrus.Entry {
 	return log.PrefixedLog("server")
 }
 
+func minTLSVersion() uint16 {
+	minTLSVer := config.GetConfig().MinTLSServeVer
+	switch minTLSVer {
+	case "1.2":
+		return tls.VersionTLS12
+	case "1.3":
+		return tls.VersionTLS13
+	default:
+		logger().Warn("Not allowed or supported mininum TLS version ", minTLSVer, ", fallback to TLS 1.3")
+
+		return tls.VersionTLS13
```

Comment:
IMO it's a bit weird/surprising that the fallback is not the same as the default specified in the struct field's annotation (1.2).

Reply:
because by default you won't need to specify it unless you don't want TLS v1.2

Reply:
Yeah I get your logic, but that only works because there's only two values for now.
Anyways, either way is fine, I don't think this is really an issue.

Reply:
Fallback to the highest (1.3) makes sense :+1: